homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -17,16 +19,27 @@
 from .const import DOMAIN
 from .coordinator import PlugwiseDataUpdateCoordinator
 from .entity import PlugwiseEntity
+from .models import PlugwiseBaseMixin
+
+T = TypeVar("T", bound=DeviceData)
 
 SEVERITIES = ["other", "info", "warning", "error"]
 
 
 @dataclass
-class PlugwiseBinarySensorEntityDescription(BinarySensorEntityDescription):
-    """Describes a Plugwise binary sensor entity."""
+class PlugwiseBinarySensorEntityDescription(
+    PlugwiseBaseMixin, BinarySensorEntityDescription
+):
+    """Describes Plugwise binary-sensor entity."""
 
+    pw_lookup: str = "binary_sensors"
     icon_off: str | None = None
 
+    def pw_get_value(self, obj: T) -> bool:  # type: ignore [override]  # pylint: disable=arguments-differ
```

Comment:
This seems like a workaround?

Reply:
Any suggestions, as it feels superfluous this way, but couldn't get mypy to just ingest from the models/override construct?

Reply:
Lets solve it when we get to sensor ... I'll remove the `@overload`s now so it is small and decent code, that will also solve this comment (for now).

Reply:
> Lets solve it when we get to sensor ... 

I don't agree. We should not add such ignores.

Reply:
That is why I removed it now from this PR, I'll have a look (and open to suggestions) for the better approach when we follow up in sensor domain. (I hate the ignores probably just as much, but kindly request some guidance to solve it, but again, removed from this PR as not necessary for the initial step with models (and/or we could forgo the models route and let every domain have it's own `pw_get_value` def-construct (but that's not keeping up with DRY).